_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+
+In terms of ordering the content, we recommend you arrange the tabs in your browser from left to right, starting with the things you would like to cover first. Avoid starting the event with very esoteric/detailed subject matter. Find something that has relevance for the whole group and is ripe for discussion. This will warm up the room and get everyone engaged. From here you can dig more deeply into highly technical protocol-specific subject matter (e.g. pull requests, BIPs, BOLTs, etc..). Once through some of the content with a high degree of technicality, you can soften the end of the meetup by covering/demoing fun projects (e.g. lightning games). We recommend using a browser plugin to save your tabs in the event your browser or browser crashes, device loses power etc.
+
+With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 
+
+You will have to balance this form of preparation with your own schedule, as there is always more to learn. Do your best to prepare, but do not over extend yourself. It is always good to remind yourself and attendees that the material is presented on a best effort basis--you are a conduit, not the source itself. Be open to making mistakes and encouraging the crowd to participate. Treat the event like the review process for open source software: stand on the shoulder of giants, don't box yourself into an island of "expertise" at the front of the room.
+
+In terms of presupposed knowledge, it is our recommendation that you communicate to your members that the basics of Bitcoin will not be explained at each event (e.g. what a hash function is). That said, when the ecosystem introduces novel concepts/schemes/mechanisms to the Bitcoin community, make sure to explain them. For example, if you are going to explain a new opcode, put the explanation in the context of preexisting opcodes and why it may or may not be an improvement to the protocol. That may require you to explain how opcodes work. Always keep in mind the perspective of the meetup members who come to each event--they play a key role in bringing life to the meetup. Find a way to quickly explain recurring themes in snippets so you do not bore people who regularly attend the events. 
```

Comment:
Preparation is important, but it's also effective when you say at the outset of a topic that you don't know a subject well and could use some help explaining it to the group. This encourages audience participation and also allows attendees to feel smart as opposed to constantly feel the information is only coming from the hosts.

Reply:
```suggestion
In terms of presupposed knowledge, it is our recommendation that you communicate to your members that the basics of Bitcoin will not be explained at each event (e.g. what a hash function is). That said, when the ecosystem introduces novel concepts/schemes/mechanisms to the Bitcoin community, make sure to explain them. For example, if you are going to explain a new opcode, put the explanation in the context of preexisting opcodes and why it may or may not be an improvement to the protocol. That may require you to explain how opcodes work. Always keep in mind the perspective of the meetup members who come to each event--they play a key role in bringing life to the meetup. Find a way to quickly explain recurring themes in snippets so you do not bore people who regularly attend the events. 

Preparation is important, but it's also effective when you say at the outset of a topic that you don't know a subject well and could use some help explaining it to the group. This encourages audience participation and also allows attendees to feel smart as opposed to constantly feel the information is only coming from the hosts.
```

Reply:
Hmm, I don't love the wording of this. "Feeling smart" seems out of character with the rest of the post. IMO what is most important is that people feel like that their opinion is welcome, not that it has any specific quality of being smart, not smart etc.